graphql_compiler/schema_generation/sqlalchemy/edge_descriptors.py

```diff
@@ -61,15 +62,26 @@ def validate_edge_descriptors(vertex_name_to_table, direct_edges):
                 )
 
 
-def generate_direct_edge_descriptors_from_foreign_keys(vertex_name_to_table):
-    """Return a set of DirectEdgeDescriptor objects from the foreign keys in the SQLAlchemy tables.
+def generate_direct_edge_descriptors_from_foreign_keys(
+    vertex_name_to_table: Dict[str, Table]
+) -> Set[DirectEdgeDescriptor]:
+    """Generate a set of edge descriptors from the foreign keys in the SQLAlchemy tables.
 
     Args:
-        vertex_name_to_table: Dict[str, Table], a mapping of vertex type names to the underlying
-                              SQLAlchemy table objects.
+        vertex_name_to_table: a mapping of vertex type names to the underlying SQLAlchemy table
+                              objects. Each SQLAlchemy table must have an unique vertex type name.
```

Comment:
Is the latter part of this true? Can we not have the same SQLAlchemy table mapped to two different vertex names?